### DISCUSSION TITLE
Simple playback via http

### DISCUSSION THREAD

**Discussion by Ghosttrap86** [MusicPlayerDaemon/MPD#1544] [Q&A]
Hi Guys,

i use an RuneAudio Player as a radio player for my bathraoom. In an old Version there were simple commands like 

`http://ipofmyruneaudio/command/?cmd=play `

Since i use MPD 0.23.5-1 these codes are not working anymore. Do i need an extra package or do i have to change a config file?

Regards  Magnus

**Comment by jcorporation** [comment]
You should ask this in the runeaudio forum.

**Reply by Ghosttrap86** [reply]
Hi Back,

thanks for your Reply. OK i thought this is an integrated feature from MPD and maybe the structure of code changed.
From RuneAudio Forum i do not get an answer... so i tried it here :-)
